src/treemodel.cpp

```diff
@@ -91,6 +91,26 @@ QVariant TreeModel::data(const QModelIndex &index, int role) const
             return item->Data(col);
             break;
         }
+        case Qt::ToolTipRole:
+        {
+            if (col == COL::Value)
+            {
+                const int ToolTipLimit = 500;
```

Comment:
I see them as three ways of viewing the value data, so they could have different amount of content:
1.  Fast single line view in tree view. The event values are strictly truncated to 300 chars, as that is more than enough to fill the column on a fairly large screen. No tail here since the tree view will cut off automatically based on the window size. 
2. This tooltip - Multi line view that can extend across whole screen, so we can increase the amount to 500 or more, with tail end of the event.
3. The value dialog. Show full content but slower to view.